parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -91,9 +91,15 @@ disableNtpAndTimesyncdInstallChrony() {
     # Disable systemd-timesyncd
```

Comment:
let's move this method to tools_install_ubuntu.sh

Reply:
might also be able to move the error constant from cse_helper.sh to tools_install_ubutu.sh.

ERR_STOP_OR_DISABLE_SYSTEMD_TIMESYNCD_TIMEOUT
ERR_STOP_OR_DISABLE_NTP_TIMEOUT
ERR_CHRONY_INSTALL_TIMEOUT